manifests/agent.pp

```diff
@@ -18,13 +18,12 @@
 class opsmatic::agent (
   $ensure               = $opsmatic::params::agent_ensure,
   $token                = $opsmatic::params::token,
-  $files_config_enabled = $opsmatic::params::files_config_enabled,
+  $host_alias           = $opsmatic::params::host_alias,
+  $filemonitorlist      = $opsmatic::params::filemonitorlist,
```

Comment:
groups is not used by the agent manifest.

It gets set by the template-erb -> https://github.com/opsmatic/puppet-opsmatic/blob/master/templates/opsmatic-agent.conf.erb